packages/logrotate/extra/dcos-logrotate-agent.service

```diff
@@ -0,0 +1,7 @@
+[Unit]
+Description=Logrotate Mesos Master: Rotate Mesos agent logs
+[Service]
+Type=simple
+EnvironmentFile=/opt/mesosphere/environment
+ExecStartPre=/usr/bin/mkdir -p /var/log/mesos/archive
```

Comment:
The agent's log directory has historically been different from the master's log directory.  I'm not sure what the history behind that is.

The helper scripts some of us use to grab logs appear to have this agent directory hard-coded.  So better not to change it.